services/brig/test/integration/API/Search.hs

```diff
@@ -41,7 +41,7 @@ testOptInOut brig = do
     refreshIndex brig
     assertSearchable "opted in" brig uid1 True
     assertCanFind brig uid2 uid1 h1
-
+    -- Even if users opt in, team users in other teams should not be found
```

Comment:
Did you mean to write a few more test lines here? Or what does this comment relate to? (maybe move to the test case that does this / delete this line)

Reply:
Ah please ignore this, it was moved to `testNonSearchableAcrossTeam`